src/components/TimelineWrapper.tsx

```diff
@@ -15,16 +19,40 @@ const TimelineWrapper: React.FC<TimelineWrapperProps> = ({
   queryKey,
   itemApiPath,
 }) => {
+  const { showSnackbar } = useContext(SnackbarContext);
+  const [submitting, setSubmitting] = useState<boolean>(false);
+
   const updatesQuery = useQuery(queryKey, async () => {
     const res = await defaultFetch(itemApiPath + '/timeline/updates');
     return handleResponseData<ZetkinUpdate[]>(res, 'GET');
   });
 
+  const notesMutation = createUseMutation(queryKey, itemApiPath + '/notes', {
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    method: 'POST',
+  })();
+
   return (
     <ZetkinQuery queries={{ updatesQuery }}>
-      {({ queries }) => <Timeline updates={queries.updatesQuery.data} />}
+      {({ queries }) => (
+        <Timeline
+          disabled={submitting}
```

Comment:
I think there is some property on `queries.updateQuery` (like `isLoading` or whatever it's called) that could be used for this to avoid having to create and maintain that state manually.

Reply:
Good shout - I hooked into `mutation.isLoading`